File: Services/Reservation_Service/Models/Reservation.cs
using System;

namespace Reservation_Service.Models
{
    public class Reservation
    {
        public int Id { get; set; }
        public int? UserId { get; set; }
        public int? PingPongTableId { get; set; }
        public DateTime Start { get; set; }
        public DateTime End { get; set; }
        public ReservationStatus ReservationStatus { get; set; }
    }
}

File: Frontend/src/app/_models/reservation.ts
export interface Reservation {
  start: Date;
  end: Date;
  pingPongTableId: number;
  pingPongTableLabel: string;
  id: number;
  userId: number;
  free: boolean;
  reservationStatus: ReservationStatus;
}
export enum ReservationStatus {
  Cancelled,
  Convirmed,
  CustomerAbsence,
  Active,
}
